src/components/sbb-radio-button/sbb-radio-button.stories.js

```diff
@@ -0,0 +1,131 @@
+import { h } from 'jsx-dom';
+import readme from './readme.md';
+
+const name = {
+  control: {
+    type: 'text',
+  },
+};
+
+const value = {
+  control: {
+    type: 'text',
+  },
+};
+
+const checked = {
+  control: {
+    type: 'boolean',
+  },
+};
+
+const disabled = {
+  control: {
+    type: 'boolean',
+  },
+};
+
+const accessibilityLabel = {
+  control: {
+    type: 'text',
+  },
+  table: {
+    category: 'Accessibility',
+  },
+};
+
+const accessibilityDescribedby = {
+  control: {
+    type: 'text',
+  },
+  table: {
+    category: 'Accessibility',
+  },
+};
+
+const accessibilityLabelledby = {
+  control: {
+    type: 'text',
+  },
+  table: {
+    category: 'Accessibility',
+  },
+};
+
+const defaultArgTypes = {
+  name: name,
```

Comment:
You can use shorthands, like:

```suggestion
  name,
  value,
  checked,
  disabled,
```